bpfd/src/lib.rs

```diff
@@ -195,15 +205,22 @@ pub async fn serve(config: Config, static_programs: Vec<StaticPrograms>) -> anyh
                 responder,
                 program_type,
             } => {
-                let prog = match program_type {
-                    command::ProgramType::Tracepoint => Program::Tracepoint(TracepointProgram {
-                        data: ProgramData::new_from_location(location, section_name, username)
-                            .await?,
-                        info: attach,
-                    }),
-                    _ => panic!("unsupported prog type"),
+                let prog_result: Result<Program, BpfdError> = match program_type {
+                    command::ProgramType::Tracepoint => {
+                        Ok(Program::Tracepoint(TracepointProgram {
+                            data: ProgramData::new_from_location(location, section_name, username)
+                                .await?,
```

Comment:
In Multi-Attach, `new_from_location()` is one level up and result stored in `prog_data_result`. Just reading code, the `?` will return if an error is found, right?

Reply:
Are you talking about the code that loads static programs?  There are at least 3 unhandled errors there.  It also only handles Xdp programs.  Did I hear someone say something about removing static program support?

In any case, that only happens at start time.  Should we skip and log an error for static program entries with problems, or just prevent bpfd from running until they fix the config?

Reply:
Never mind.  I see it now.  I'll take care of that one too.

Reply:
Done